kotlinx-coroutines-core/native/src/Dispatchers.native.kt

```diff
@@ -12,6 +12,11 @@ public actual object Dispatchers {
     public actual val Default: CoroutineDispatcher get() = DefaultDispatcher
     public actual val Main: MainCoroutineDispatcher = createMainDispatcher(Default)
     public actual val Unconfined: CoroutineDispatcher get() = kotlinx.coroutines.Unconfined // Avoid freezing
+
+    @ExperimentalCoroutinesApi
+    public fun CoroutineDispatcher.shutdown() {
```

Comment:
Please add documentation to this method.

Also, it's probably worth adding it right to common `Dispatchers` (but with effect only on Native problem) to simplify adoption for our MPP users